docs/library/uos.rst

```diff
@@ -294,5 +302,7 @@ that the block device supports the extended interface.
             (*arg* is unused)
           - 6 -- erase a block, *arg* is the block number to erase
 
-See :ref:`filesystem` for example implementations of block devices using both
-protocols.
+Unless otherwise stated ``ioctl(op, arg)`` can return None. Consequently an
+implementation can ignore unused values of ``op``. As a minimum
+``ioctl(4, ...)`` must be intercepted; for littlefs ``ioctl(6, ...)`` must be
+intercepted and return 0. The need for others is hardware dependent.
```

Comment:
Perhaps this para would be better indented so it's part of the ioctl method description?

So the intended semantics for the return value is:
- if it's not implemented then return `None` (so unused `op`'s can be safely ignored, as you point out)
- 4 and 5 should return ints (as already documented)
- others, if implemented, should return 0 for success and non-zero for failure, with the value returned being an OSError errno code

Reply:
Thank you for that. I've now updated the doc.